### DISCUSSION TITLE
[V5] how to force adonis send cors header on static file request

### DISCUSSION THREAD

**Discussion by khaeroen** [adonisjs/core#2259] [Help] [answered]
intro:
I need to get uploaded files from my other adonis site.. but i get error cors..

> Access to fetch at 'http://127.0.0.1:52447/uploads/qrcode2.png' from origin 'http://localhost:3333' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

in adonis 4, there is similar topic:
[https://forum.adonisjs.com/t/how-to-force-adonis-send-cors-headers-on-static-file-requests/3989]

but i don't have any idea how to implement that solution in adonis 5.. when i tried to access dynamic route as usual, there isn't any cors error, it's just success..

**Comment by RomainLanz** [comment, marked as answer]
Hey @khaeroen! 👋 

[This was a bug](https://github.com/adonisjs/core/issues/2156) in the `HTTP Server`, it has been fixed and should work as expected in the next release.

**Reply by khaeroen** [reply]
oh, ok.. i see.. thanks for the information.. 
for now, i just create a route method post that accept path for the file.. and then in controller, i use something like this:

`// get filePath from request`
`const file = fs.readFileSync(Application.makePath(filePath))`
`return file`

now there isn't any error cors..
